.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+**Screenshots**
+
+**Related Issue**
+E.g: *Closes/Fixes #1234*
+
+- [ ]  **Code Quality**
+    - [ ]  The code follows established coding standards and style guidelines
+    - [ ]  The code is properly commented on and provides context when needed (complex feature or hard to understand)
+- [ ]  **Functionality validation**
+    - [ ]  Feature working as described in the related issue
+    - [ ]  Does this PR need environment variables to be set? If yes, have they been set?
+    - [ ]  New API endpoints are properly documented
+    - [ ]  New functions are properly tested (When should we add tests?)
+    - [ ]  Code errors are being logged in the code health service
+- [ ]  **Product validation**
+    - [ ]  UI changes look the same as in Figma
+    - [ ]  Usage events are being sent to the product analytics service
```

Comment:
The usage events we want to register should be defined at the issue.

It would be nice to have an Issue definition checklist too!

Reply:
Agree with this and will work on it. But besides having the definition at the issue level, this checklist should validate that the changes were actually made.

Reply:
https://github.com/decentraland/governance/issues/1372 WIP